README.md

```diff
@@ -106,7 +106,7 @@ We could have also used `unfoldr`:
 ```Java
 Iterable<Integer> positiveSquaresBelow100 = unfoldr(x -> {
               int square = x * x;
-              return square < 100 ? Optional.of(tuple(square, x + 1)) : Optional.empty();
+              return square < 100 ? Maybe.maybe(tuple(square, x + 1)) : Maybe.nothing();
```

Comment:
Let's use `Maybe.just` here for the positive case in the ternary switch; that's analogous to `Optional.of`